admin_app/src/app/page.tsx

```diff
@@ -146,6 +156,20 @@ export default function Home() {
       : setNewCardData({
           content_title: newCardData.content_title,
           content_text: content_text,
+          content_language: newCardData.content_language,
+        });
+  };
+
+  const onContentLanguageChange = (content_language: string) => {
```

Comment:
Only on change do we change it to the new language. If you don't change the language, `setCardToEdit` is left with the empty string as the language